packages/lit-dev-tools-esm/src/fake-github-middleware.ts

```diff
@@ -394,4 +401,71 @@ class FakeGitHub {
     this._gists.set(gist.id, gist);
     return jsonResponse(ctx, 200, gist);
   }
+
+  /**
+   * Simulates PATCH https://api.github.com/gists/<id>, the API for creating a
+   * new revision of an existing GitHub gist.
+   *
+   * Documentation:
+   * https://docs.github.com/en/rest/reference/gists#update-a-gist
+   */
+  async updateGist(ctx: Koa.Context) {
+    ctx.set('Access-Control-Allow-Origin', '*');
+    const accept = ctx.get('accept');
+    if (accept !== 'application/vnd.github.v3+json') {
```

Comment:
Nit: Just a mock so not required. But these initial validation functions are the same across each endpoint so could be separated.

Reply:
Done